samples/IISSample/Startup.cs

```diff
@@ -12,6 +13,7 @@ public class Startup
     {
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddAuthentication();
```

Comment:
Windows auth doesn't really need the AuthN middleware, it could do it itself / resolve and invoke the service directly.